Data/ByteString/Internal.hs

```diff
@@ -758,3 +759,6 @@ foreign import ccall unsafe "static fpstring.h fps_minimum" c_minimum
 
 foreign import ccall unsafe "static fpstring.h fps_count" c_count
     :: Ptr Word8 -> CULong -> Word8 -> IO CULong
+
+foreign import ccall unsafe "static fpstring.h fps_sort" c_sort
+    :: Ptr Word8 -> CULong -> IO ()
```

Comment:
Why `CULong` and not `CSize`?  (With corresponding `size_t` in the C prototype)?
```suggestion
    :: Ptr Word8 -> CSize -> IO ()
```

Reply:
Other functions above use `CULong`. Do you think it is worth to change them as well?

Reply:
> Other functions above use `CULong`. Do you think it is worth to change them as well?

Yes, that'd be the correct signature.  Most platforms these days have `unsigned long ~ size_t`, but the latter is the more correct choice. Of course the current definition matches the `C` prototypes, so the code "works" as-is, but it should be using `size_t` instead.  Perhaps there used to be Windows versions where size_t wasn't known to the compiler? I would not expect that to be a problem now.

It may be prudent to do a build on Windows though before cutting an official release.